All In/Features/Profile/ProfileView.swift

```diff
@@ -92,6 +95,35 @@ struct ProfileView: View {
         }
     }
 
+    private var photoPicker: some View {
+        VStack {
+            PhotosPicker(
+                selection: $imageSelection,
+                matching: .images
+            ) {
+                image
+                    .resizable()
+                    .scaledToFill()
+                    .frame(width: 125, height: 125)
+                    .background(.white)
+                    .cornerRadius(100)
+                    .shadow(color: .black.opacity(0.25), radius: 2)
+            }
+            .labelsHidden()
+        }
+        .onChange(of: imageSelection) { newItem in
+            guard let newItem = newItem else { return }
+
+            Task {
+                if let loaded = try? await newItem.loadTransferable(type: Image.self) {
+                    self.image = loaded
+                } else {
+                    print("Failed to edit profile image")
```

Comment:
Can you try setting up a logger to log errors rather than print. We should try to shift away from print statements.

You can create a logger class with a shared instance, and create separate loggers for each issue:

```
import os

let viewsLogger = Logger(subsystem: Bundle.main.bundleIdentifier ?? "com.cornellappdev.allin", category: "Views")
```